core/domain/user_jobs_one_off_test.py

```diff
@@ -1354,6 +1354,31 @@ def test_that_last_edited_and_created_time_are_not_updated(self):
         self.assertIsNone(owner_settings.last_edited_an_exploration)
 
 
+class FillExplorationIdsInUserSubscriptionsModelOneOffJobTests(test_utils.GenericTestBase): # pylint: disable=line-too-long
```

Comment:
Try to break the lines instead of using `# pylint: disable=line-too-long`
```suggestion
class FillExplorationIdsInUserSubscriptionsModelOneOffJobTests(
    test_utils.GenericTestBase):
```

Reply:
Done.